package-res/ccc/core/base/color.js

```diff
@@ -208,7 +208,7 @@ def
        
         var min = extent.min.value,
             max = extent.max.value;
-        
+        if (max >= Math.pow(2,53)) { max = max / Math.pow(2,53); min = max; }
```

Comment:
See notes below on the protovis.js file. The same applies here, w.r.t. to clamping and creating a variable to hold the magic value.
Also, individually clamp each of `min` and `max`.